deploy/manifests/dynatrace-service/dynatrace-service.yaml

```diff
@@ -324,11 +323,26 @@ spec:
                 secretKeyRef:
                   name: dynatrace
                   key: DT_API_TOKEN
+            - name: DT_PAAS_TOKEN
```

Comment:
Not sure, but I think @grabnerandi has removed the PAAS Token. Do we need it again?

Reply:
good catch, that one can be removed again